### DISCUSSION TITLE
Possible issue in quick-start tutorial

### DISCUSSION THREAD

**Discussion by Mr-Draco** [reduxjs/redux-toolkit#2201] [Q&A] [answered]
I went through the [quick-start tutorial](https://redux-toolkit.js.org/tutorials/quick-start). When I finished, the project would not render because state.counter was undefined ( while trying to read state.counter.value ). Using ```const count = useSelector((state) => { console.log(state); return state.counter.value; } )```, I could see that the state was in fact ```{ value: 0 }```, instead of ```{ counter: { value: 0 } }```. I changed the `useSelector` statement to ```const count = useSelector((state) => state.value)``` and it worked. However, it seems useful to group data by the slice... so I'm wondering if there is a way to have the state use the name from createSlice. 

It is possible that I made a mistake, but wanted to see if this happened for anyone else. Can anyone confirm the above?

**Comment by markerikson** [comment, marked as answer]
@MrDracoG Can you show your `configureStore` call?  That sounds like you may have done `reducer: counterReducer`, when it _should_ be `reducer: {counter: counterReducer}`.

**Reply by Mr-Draco** [reply]
Ahh, that seems right!

**Reply by Mr-Draco** [reply]
I knew I was goofing something up. Thank you.
